plugin/pkg/scheduler/factory/factory.go

```diff
@@ -517,7 +517,11 @@ func (factory *ConfigFactory) MakeDefaultErrorFunc(backoff *util.PodBackoff, pod
 		if err == core.ErrNoNodesAvailable {
 			glog.V(4).Infof("Unable to schedule %v %v: no nodes are registered to the cluster; waiting", pod.Namespace, pod.Name)
 		} else {
-			glog.Errorf("Error scheduling %v %v: %v; retrying", pod.Namespace, pod.Name, err)
+			if _, ok := err.(*core.FitError); ok {
+				glog.V(4).Infof("Unable to schedule %v %v: no fit: %v; waiting", pod.Namespace, pod.Name, err)
```

Comment:
well, my suggestion was to keep it as noisy as glog.Errorf (to keep it same from output perspective), with the advantage that it wont be printed to stderr by default what glog does. perhaps just to address the comment by @davidopp. 

That said, I completely agree with this current change too and comments by others.